core/arch/emit_utils_shared.c

```diff
@@ -3557,8 +3557,10 @@ create_syscall_instr(dcontext_t *dcontext)
         return create_int_syscall_instr(dcontext);
     } else if (method == SYSCALL_METHOD_SYSENTER) {
         return INSTR_CREATE_sysenter(dcontext);
+#    ifdef X64
```

Comment:
Maybe add this change to the PR description explicitly?

Reply:
Done.  Actually -- I think we can revert this, since this is valid for 32-bit AMD.